app/packages/utilities/src/index.ts

```diff
@@ -354,6 +354,15 @@ const isURL = (() => {
   const nonLocalhostDomainRE = /^[^\s\.]+\.\S{2,}$/;
 
   return (string) => {
+    return true;
```

Comment:
I think it would be best to only treat `^http[s]://` and perhaps `^localhost:`and `^0.0.0.0:` as URLs, since users could provide all of the following:

```
s3://bucket/path
gs://bucket/path
[custom-minio-alias]://bucket/path
```

Reply:
I defer to you on how to finalize, but avoiding linking `[custom-minio-alias]://bucket/path` while still linking other things seems ill-defined

Reply:
If you don't want to think about it right now, also fine punting. The currently implemented behavior covers the 90% case

Reply:
Apologies, I overlooked. Will address tomorrow